pallets/committee/src/types.rs

```diff
@@ -189,7 +189,20 @@ impl<O: Into<Result<Origin<T>, O>> + From<Origin<T>> + Clone, T: Config> EnsureO
 
 	#[cfg(feature = "runtime-benchmarks")]
 	fn successful_origin() -> O {
-		O::from(CommitteeOrigin::ApprovedByCommittee(Default::default(), Default::default()))
+		use frame_benchmarking::vec;
+		O::from(CommitteeOrigin::ApprovedByCommittee(
+			Default::default(),
+			VoteAggregate {
+				votes: vec![
+					MemberVote {
+						member: CommitteeMember { account_id: Default::default(), member_type: MemberType::Council },
+						vote: Vote::Aye
+					};
+					T::MinCouncilVotes::get() + 1
+				],
+				end: <frame_system::Pallet<T>>::block_number() + 1_u32.into(),
+			},
```

Comment:
Can you please add a quick test that checks that is indeed successful?

Reply:
emmm

This seems only can be tested with benchmarks =.=, we can mock an OuterOrigin to test this but only in tests

This modification works on my local machine to run benchmarks, for testing this, I think the better way is to write a CI try executing benchmarks in our repo

Reply:
you're right, since it works in the benchmarks already, should be fine